github/ci/testgrid/gen-config.yaml

```diff
@@ -5,7 +5,19 @@ dashboards:
 - name: kubevirt-libguestfs-appliance-periodics
 - name: kubevirt-hyperconverged-cluster-operator-presubmits
 - name: kubevirt-hyperconverged-cluster-operator-periodics
-
+- name: kubevirt-hyperconverged-cluster-operator-openshift-ci-presubmits
+  dashboard_tab:
+    - name: pull-ci-kubevirt-hyperconverged-cluster-operator-main-ci-index
```

Comment:
Could you please decrease the indentation of the array element 2 spaces? see how other array elements are indented in this file